app/javascript/components/components/Header.js

```diff
@@ -1,47 +1,56 @@
-import React, { Component } from 'react'
-import { Nav, NavItem } from 'reactstrap'
-import { NavLink } from 'react-router-dom'
-
+import React, { Component } from "react";
+import { Nav, NavItem } from "reactstrap";
+import { NavLink } from "react-router-dom";
 
 export default class Header extends Component {
   render() {
-    const {
-      logged_in,
-      current_user
-    } = this.props
+    const { logged_in, current_user } = this.props;
 
-    return(
+    return (
       <>
         <h1>Why Stress When You Can Wng It?</h1>
         <Nav className="nav">
+          <NavItem className="navitem">
+            <NavLink to="/reviewindex" className="nav-link">
+              <img
+                className="logo-home-button"
+                src={require("../assets/logo-favicon-headernav.png")}
+              />
+            </NavLink>
+          </NavItem>
+          <NavItem className="navitem">
+            <NavLink to="/aboutus" className="nav-link">
+              About Wng-N-It
+            </NavLink>
+          </NavItem>
+          <NavItem className="navitem">
+            <NavLink to="/reviewindex" className="nav-link">
+              Wng Feed
+            </NavLink>
+          </NavItem>
+          {logged_in && (
             <NavItem className="navitem">
-              <NavLink to="/reviewindex" className="nav-link">
-                <img className="logo-home-button" src={require('../assets/logo-favicon-headernav.png')}/>
-              </NavLink>
-            </NavItem>
-            <NavItem className="navitem">
-              <NavLink to="/aboutus" className="nav-link">About Wng-N-It</NavLink>
-            </NavItem>
-            <NavItem className="navitem">
-              <NavLink to="/reviewindex" className="nav-link">Wng Feed</NavLink>
-            </NavItem>
-          {logged_in &&
-            <NavItem className="navitem">
-              <a href='/users/sign_out' className="nav-link">Sign Out</a>
+              <a href="/users/sign_out" className="nav-link">
```

Comment:
Since you are passing in Devise routes to your header you can reference the variable that holds the route. `sign_out_route`